### ISSUE TITLE
The Inspector displays incorrect rectangle for the ListViewGroup

### ISSUE THREAD

**Issue by SergeySmirnov-Akvelon** [dotnet/winforms#4778]
.NET Core Version: 
- 6.0.100-preview.2.21155.3

Have you experienced this same bug with .NET Framework?: 
- No (regression from #3224)

**Minimal repro:**
1. Create an app with a ListView
2. Add ListViewGroup to the ListView
3. Add ListViewItem related to the group to the ListView
4. Run the application
5. Run the inspect
6. Select a ListViewGroup in the Inspect tree

**Actual behavior:**
The ListViewGroup has an incorrect rectangle
![Issue-4778-actual](https://user-images.githubusercontent.com/23376742/114395291-da0a9300-9ba4-11eb-850e-bb2e5c03ab59.png)

**Expected behavior:**
The ListViewGroup should have a correct rectangle
![Issue-4778-expected](https://user-images.githubusercontent.com/23376742/114395310-decf4700-9ba4-11eb-980d-48b6a4b4b2e8.png)

**Comment by Zheng-Li01** [comment]
Verified the issue with latest .NET 6.0.100-preview.5.21228.19, the issue has been fixed that the inspect tool display correct rectangle for ListViewGroup for now, see below GIF.
![Verified4778](https://user-images.githubusercontent.com/38325459/116525419-152df580-a90b-11eb-8d14-873d23b7838b.gif)
